kolibri/plugins/facility/assets/src/constants.js

```diff
@@ -10,8 +10,6 @@ export const PageNames = {
   DATA_EXPORT_PAGE: 'DATA_EXPORT_PAGE',
   FACILITY_CONFIG_PAGE: 'FACILITY_CONFIG_PAGE',
   ALL_FACILITIES_PAGE: 'ALL_FACILITIES_PAGE',
-  MANAGE_SYNC_SCHEDULE: 'MANAGE_SYNC_SCHEDULE',
```

Comment:
Why are we removing these pages from the facility? We want to be able to manage sync schedules in both the facility _and_ the device plugins.